### ISSUE TITLE
onCreateCommand: local user not correctly set-up when building devcontainer

### ISSUE THREAD

**Issue by jzazo** [loft-sh/devpod#1173]
**What happened?**  
When I build a devcontainer image and it runs the `onCreateCommand`, it does not setup the local user properly. I get a warning when installing pip:
```
error WARNING: The scripts pip, pip3 and pip3.10 are installed in '/home/vscode/.local/bin' which is not on PATH.
```
It seems devpod is not sourcing the `.profile` file correctly when running the `onCreateCommand` script. I checked and `/home/vscode/.profile` file exists.

**What did you expect to happen instead?**  
That `/home/vscode/.profile` is sourced when running the `onCreateCommand` script.

**Local Environment:**  
- DevPod Version: v0.5.16
- Operating System: windows
- ARCH of the OS: AMD64

**DevPod Provider:**  
- Local/remote provider: ssh

**Comment by pascalbreuninger** [comment]
Hi @jzazo, thanks for reporting the issue. You've hit an unimplemented property of the devcontainer.json spec. 
We'll implement it going forward

**Comment by pascalbreuninger** [comment]
@jzazo this has been implemented with #1184 and will be available with the next release 👍
